src/storage/btree.c

```diff
@@ -14603,12 +14588,11 @@ btree_find_AR_sampling_leaf (THREAD_ENTRY * thread_p, BTID * btid, VPID * pg_vpi
   int est_page_size, free_space;
   int key_cnt = 0;
   int root_level = 0, depth = 0;
-  double prob = 1.0;		/* Acceptance probability */
 
   assert (stat_info_p != NULL);
   assert (found_p != NULL);
 
-  *found_p = false;		/* init */
+  *found_p = true;		/* init */
```

Comment:
It is trivial.

`btree_find_AR_sampling_leaf` is only called by `btree_get_stats_with_AR_sampling`.
How about not using `found` in `btree_get_stats_with_AR_sampling` and `btree_find_AR_sampling_leaf`?

Reply:
Okay. I'll fix it on next PR.